guide/case-studies/wallet-scrutiny.md

```diff
@@ -0,0 +1,215 @@
+
+- **Consider the users goals**: Understanding the user groups who would visit the website helped us to understand the shortest user journey that would help them to meet their main goals. This helped with the information architecture.
+
+- **Trust**: Creating trust throughout the user experience was something that we tried to incorporate using design. This was essential to keeping the user engaged and trusting the reviews.
+
+## Conclusion
+As a community we really enjoyed working with this project and sprinting towards the redesign of the website. The entire process took about six months in total and we jumped on weekly or bi-weekly calls to keep up the momentum. The redesign of a website is an intricate process that requires a team effort as well as understanding the needs and goals of the people using the website.
+
+With that said this project is not finished. While we made a lot of changes in the redesign process the next steps with this project are to get the project in front of the world, gather feedback and speak with users and continue to iterate. There is more work to be done on the expert opinions, how the project could integrate Nostr and so contributions to that effort would be much appreciated.
+
+## Acknowledgements/Gratitude
+We would like to thank the WalletScrutiny team for their willingness to collaborate as well as members of the Bitcoin Design Community, who rallied together to make the redesign happen. Christoph and Mogashni worked with the team over a longer period of time to make the redesign a reality. We’d also like to give a shout out to our anonymous copy guy. He spent a lot of time rewriting all content for length and clarity and came up with some headlines that really capture what Wallet Scrutiny does, while still being catchy and approachable.
+
+Moving forward we will continue to support the project in order to bring more awareness to wallet reproducibility in the form of education as well as support in the design.
+
+## Resources
+Previous case studies:
+- [Blixt wallet](https://bitcoin.design/guide/case-studies/blixt-wallet/)
+- [Payjoin](https://bitcoin.design/guide/case-studies/payjoin/)
+
+Links to project resources:
+- [Figma file](https://www.figma.com/file/afzvWhwhxIwhoMans8lQg9/Wallet-Scrutiny?type=design&node-id=1690%3A71714&mode=design&t=rHRO9up7hGZqM3gf-1)
+- [Case Study Guidelines](https://docs.google.com/document/d/1lkipBAdo-re_TCIXgMhddFpGnZU3u6f1svJ06ctjkrk/edit#heading=h.rpf7ryqjg7i5)
+- [Case Study Guideline questions](https://docs.google.com/document/d/1B6qcry5Wb-jjtMpTyuTk5WqGvF8WKScc4NAIK-SK1go/edit)
+- [Call recordings](https://www.youtube.com/playlist?list=PLpV0KfVOMojbYu2i1zRjGcJw9fA1cj0eY)
+
+
+---
+
+Next, view the [resources]({{ '/guide/resources' | relative_url }}) a collection of design resources to that can help you design and build better bitcoin products.
+
+{% include next-previous.html
+   previousUrl = "/guide/case-studies"
```

Comment:
Should link to the Payjoin case study, which is the previous page in the nav structure.